<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModiftRoomPrice</class>
 <widget class="QWidget" name="ModiftRoomPrice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="title">
          <string>房间信息</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTableView" name="tableView"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">image: url(:/image/house/1.jpg);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="1" column="1">
               <widget class="QLineEdit" name="lineEditName">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="placeholderText">
                 <string>请输入名称</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_3">
                <property name="font">
                 <font>
                  <family>楷体</family>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>价格：</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="lineEditPrice">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="placeholderText">
                 <string>请输入价格</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="font">
                 <font>
                  <family>楷体</family>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>房间类型编号：</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEditNo">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>40</width>
                  <height>26</height>
                 </size>
                </property>
                <property name="placeholderText">
                 <string>请输入类型编号</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_2">
                <property name="font">
                 <font>
                  <family>楷体</family>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>名称：</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="label_4">
                <property name="font">
                 <font>
                  <family>楷体</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>类型编号从 1 开始</string>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QLabel" name="label_5">
                <property name="font">
                 <font>
                  <family>楷体</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>房间价格单位：元</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pbnAdd">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>楷体</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>OpenHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>添加</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbnModify">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>楷体</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>OpenHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>修改</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbnSave">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>楷体</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>OpenHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>保存</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbnCancel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>23</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>楷体</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>OpenHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>取消</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
